.vscode/settings.json

```diff
@@ -8,5 +8,8 @@
             "source.organizeImports.ruff": "explicit"
         }
     },
-    "python.testing.pytestEnabled": true
+    "python.testing.pytestEnabled": true,
+    "python.analysis.extraPaths": [
```

Comment:
minor add for VSCode, it had to look for schemas too as it appeared as a separate dependency